cmd/cmd.go

```diff
@@ -34,10 +33,47 @@ import (
 	"github.com/uber-common/bark"
 	"github.com/uber/aresdb/clients"
 	"github.com/uber/aresdb/memutils"
+	"github.com/spf13/cobra"
 )
 
-// StartService is the entry point of starting ares.
-func StartService(cfg common.AresServerConfig, logger bark.Logger, queryLogger bark.Logger, metricsCfg common.Metrics, httpWrappers ...utils.HTTPHandlerWrapper) {
+// AppParams contains application parameters for creating command
+type AppParams struct {
+	DefaultCfg   map[string]interface{}
+	ServerLogger bark.Logger
```

Comment:
maybe we can enhance this by checking ServerLogger, QueryLogger, Metrics is nil or not and set default accordingly

Reply:
Done